app/assets/stylesheets/moderators.scss

```diff
@@ -83,8 +83,8 @@
   width: 96%;
   max-width: 1600px;
   border-radius: 3px;
-  border: 1px solid var(--container-color-tertiary);
-  background: var(--container-bg);
+  border: 1px solid var(--card-color-tertiary);
+  background: var(--card-bg);
```

Comment:
I'm not sure if it's me or not but I checked this PR locally and something is off: you're referencing new CSS variables that are not yet available in your branch, for example all `--card-*` variables that you're referencing in `moderators.scss` do not defined in your code (all color variables are defined in [_colors.scss](https://github.com/catalinpit/dev.to/blob/catalinpit/fix-mod-colors-5738/app/assets/stylesheets/config/_colors.scss) file). BUT if you check the same file in our repo on master branch, these variables [do exist](https://github.com/thepracticaldev/dev.to/blob/master/app/assets/stylesheets/config/_colors.scss#L61-L67). These have been introduced very recently, so I'm pretty sure it will be ok once you merge master in.

Reply:
I didn't change that. There are more card changes which I didn't make in my branch.

What is happening? Did master got merged into my branch or something?

Should I re-do the PR? I didn't include those for sure. You can have a look at my first commit.